testplan/web_ui/testing/package.json

```diff
@@ -43,7 +43,7 @@
     "react-scripts": "^3.4.1",
     "react-spinners": "^0.6.0",
     "react-syntax-highlighter": "^11.0.2",
-    "react-test-renderer": "16.6.0",
+    "react-test-renderer": "16.8.5",
```

Comment:
This update is needed to allow the shallow method of enzyme to satisfy the Rule of Hooks with useState hook.